Source/CronosPlayUnreal/Android/SecureStorage.java

```diff
@@ -0,0 +1,87 @@
+package com.cronos.play;
+
+import android.content.Context;
+import android.util.Log;
+import androidx.security.crypto.EncryptedFile;
+import androidx.security.crypto.MasterKey;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.FileOutputStream;
+
+public class SecureStorage {
```

Comment:
should read and write have a `finally:` block that will make sure to close the file/stream in any case (success or exception)?

Reply:
finally seems good, i'll change